src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -2435,6 +2435,7 @@ void mvnw(Register Rd, Register Rm,
       break;
     default:
       ShouldNotReachHere();
+      Rm = 0;  // unreachable
```

Comment:
Instead of adding this line, how about initiating `Rm` at declaration like `int Rm = 0;`?

Reply:
This is how it's done elsewhere in the port. Sucks, but we're stuck with it unil we find a better solution for warnings.